layouts-api-react/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Make sure you update the Readme to describe your entry. You should provide some instructions for how to run the application (Installing requirements, starting the server) and how to use the application once it's running.

Reply:
Seems like my last push didn't go through before I made the pull request. The "Add Pane" and "Remove Panes" buttons have related functions and work in the version I have so that has to be the issue here. I also had set up a .env to hold variables before I made the last push, but it wouldn't accept the .env in GitHub with the push. The README.md I can work on to include dependency installations. However, instructions for running the React app itself are in the default README.md that is made when you first create the react app. I can include that in the README.md as well to clarify this. Sorry for not paying attention to this by the way I had some other stuff going on.